bluepyefe/reader.py

```diff
@@ -208,16 +208,16 @@ def csv_lccr_reader(in_data):
         in_data (dict): of the format
         {
             'filepath': "./XXX.txt",
-            'dt': 1,
+            'dt': 0.1,
             'ton': 2000,
             'toff': 2500,
             'ljp': 14.0,
             'amplitudes': [10 -10 20 -20 30 -30 40 -40 50 -50],
-            'hypamp': -0.2,
+            'hypamp': -20 (units should match 'amplitudes'),
```

Comment:
Could you put a # before the units comment, so that this would be a valid dict in python, i.e.
`'hypamp': -20, #  (units should match 'amplitudes')`